### ISSUE TITLE
Travis for OSX

### ISSUE THREAD

**Issue by Maijin** [hteso/iaito#64]

**Comment by hteso** [comment]
Can't authorize yet, I have to solve some issues first related to my account. I'll do it ASAP.

**Comment by Maijin** [comment]
It will need @probonopd to configure travis.yml for this too

**Comment by hteso** [comment]
Travis is now working, AFAIK for linux only. What do you need to make it work for OS X?

**Comment by Maijin** [comment]
See https://github.com/radare/radare2/blob/master/.travis.yml#L5 but probably need a different script.

**Comment by probonopd** [comment]
Sorry I don't know how to do it for macOS, but should be documented on the Travis CI site.

**Comment by Maijin** [comment]
![image](https://cloud.githubusercontent.com/assets/1408600/24770921/e7dede6e-1b0b-11e7-9cee-30fd9d39c6e9.png)

**Comment by probonopd** [comment]
That will most likely not be the only change.

**Comment by Maijin** [comment]
that's the only change required to enable travis for OSX.
For the rest it's all about the script

**Comment by ballessay** [comment]
Yes it's mostly about the script but there will be other tasks to solve.
It seems one needs to install Qt by homebrew, but a quick search in [homebrew-core](https://github.com/Homebrew/homebrew-core/find/master) revealed the only available versions are 5..5, 5.7 and 5.8. (note: Doesn't brew install from source? How long does a Qt compile take no travis?)
Support for a 'make install' target that copies all the needed stuff into the app_bundle.

Some links for later reference:
https://docs.travis-ci.com/user/multi-os/
https://docs.travis-ci.com/user/osx-ci-environment/
https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-OSX

Example:

**Comment by Maijin** [comment]
![image](https://cloud.githubusercontent.com/assets/1408600/24839993/32bd1166-1d65-11e7-97d9-a3ec7b4cd7c3.png)
From @mrexodia

**Comment by ylluminate** [comment]
Any luck on this so far as I would also like to simply see the option to be able to install `iaito` via `brew install iaito` since I think it would help a lot of folks participate more readily in the dev and testing.
